components/alert/alert.js

```diff
@@ -0,0 +1,156 @@
+import '../button/button-icon.js';
+import '../button/button-subtle.js';
+import '../colors/colors.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { bodyCompactStyles } from '../typography/styles.js';
+import { LocalizeStaticMixin } from '../../mixins/localize-static-mixin.js';
+import { RtlMixin } from '../../mixins/rtl-mixin.js';
+
+class Alert extends LocalizeStaticMixin(RtlMixin(LitElement)) {
+
+	static get properties() {
+		return {
+			buttonText: { type: String, attribute: 'button-text' },
+			hasCloseButton: { type: Boolean, attribute: 'has-close-button' },
+			hidden: { type: Boolean, reflect: true },
+			subtext: { type: String },
+			type: { type: String, reflect: true }
+		};
+	}
+	static get styles() {
+		return [bodyCompactStyles, css`
+
+			:host {
+				animation: 600ms ease drop-in;
+				background: white;
+				border-radius: 0.3rem;
+				border: 1px solid var(--d2l-color-mica);
+				box-sizing: border-box;
+				display: block;
+				display: flex;
```

Comment:
I don't think `display: block` is supposed to be there? I don't see it on the old version.